lib/veryfi/api/document.rb

```diff
@@ -39,11 +39,9 @@ def process(raw_params)
         params = setup_create_params(raw_params)
 
         file_content = File.read(params[:file_path])
-        file_name = File.basename(params[:file_path], ".*")
```

Comment:
Let's check if it has an extension and don't use otherwise. Extensions helps to recognize base64, as not all people include it in the hash.